Ally4/Portfolio
# Admin`s navigation bar, access and control

# Description

This PR contains the navigation bar, access, and control for the admin

## Type of change

Please check the options that are relevant.


# How should this be manually tested or reproduced?

- Clone this repo
- Checkout to `ft-admin`
- Run the command `npm i`
- Run the command `npm start` to run the  system locally 
- Write localhost:3000/admin to access the admin`s navigation bar, access, and control (as the authentication is yet settled)
- Navigate through the admin`s landing page and other pages to see the difference.

# Checklist:

- [X] Creating an admin`s navigation bar
- [X] Creating the home component for the admin
- [X] Creating the blog component for the admin
- [X] Creating buttons to perform admin`s role
- [X] Creating an action component to perform admin`s tasks
- [X] Creating the create blog component, for the admin to create a blog
- [X] Styling the created features
